requirements-top-level.txt

```diff
@@ -5,7 +5,7 @@ autopep8
 backoff
 bandit
 boto3
-boxsdk[jwt]==2.0.0a11
+boxsdk[jwt]==2.6.0
```

Comment:
untag this here please. it does not need to be pinned here now that we're off of alpha and can use the 'latest' release